sentry/src/main/java/io/sentry/InvalidSentryTraceHeaderException.java

```diff
@@ -0,0 +1,17 @@
+package io.sentry;
+
+import org.jetbrains.annotations.NotNull;
+
+public final class InvalidSentryTraceHeaderException extends Exception {
```

Comment:
should we add java docs if this is public or make it package private?

Reply:
I'll add javadocs, we can't make it package private as it can be thrown in the client code 👍